apps/fortuna/src/config.rs

```diff
@@ -66,6 +68,9 @@ pub enum Options {
     /// Request a random number from the contract.
     RequestRandomness(RequestRandomnessOptions),
 
+    /// Inspect recent requests and find unfulfilled requests with callback.
+    Inspect(InspectOptions),
```

Comment:
question about this: how do I tell that the request didn't get a callback from the result? It seems like the output simply prints out all of the requests that have happened.

Reply:
any request still stored in the contract is not fulfilled yet. We just iterate through the requests and find the ones where the callback flag is also true and print them out.